tests/test_asttokens.py

```diff
@@ -80,6 +80,8 @@ def test_token_methods(self):
     self.assertTrue(token.ISEOF(atok.find_token(atok.tokens[5], tokenize.NL).type))
 
   def test_unicode_offsets(self):
+    if six.PY2:
+      pytest.skip("Unicode parsing is different on Python 2")    
```

Comment:
No idea how anyone got this working on 2.7. This should have failed before now! See also https://github.com/gristlabs/asttokens/runs/4493079453?check_suite_focus=true